### PR TITLE
Objects Automated Test - LPS-141676 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141676

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/CreateObject.macro [hunk 1]
```diff
@@ -40,6 +40,12 @@ definition {
 			locator1 = "CreateObject#OBJECT_NAME");
 	}
 
+	macro assertCustomObjectNotPresent {
+		AssertTextNotEquals(
+			locator1 = "ObjectPortlet#SEARCH_RELATIONSHIP_ENTRY",
+			value1 = "${customObject}");
+	}
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/CreateObject.macro:47]: Please, you can change the name of the macro for `assertEntryOnRelationshipFieldNotPresent` and change the value `customObject` for `entryValue`. 
Please, change the file of macro for `ObjectPortlet.macro`

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias ,

Please, check my comments and let me know if you have any questions.

Thanks.

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**